client/src/components/CinemaHall/index.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable no-plusplus */
+import React, { useContext } from 'react';
+import Seat from '../Seat';
+import './style.scss';
+import ReservationProvider from '../../context/Reservation';
+
+const CinemaHall = () => {
+  const nrOfRows = 7;
+  const nrOfColumns = 8;
+  const columns = [];
+  const rowsInLetter = [];
+
+  for (let i = 1; i <= nrOfColumns; i++) {
+    columns.push(i);
+  }
+  for (let j = 1; j <= nrOfRows; j++) {
+    rowsInLetter.push(String.fromCharCode(j + 64));
+  }
+
+  return (
+    <ReservationProvider>
```

Comment:
It might be provided in the app because we will be setting it when the user clicks on the starting time of the show. It will be later user in the next screens.